src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -0,0 +1,171 @@
+    for organization in RangeQuerySetWrapper(
+        organizations, step=10000, result_value_getter=lambda item: item.id
+    ):
+        if features.has("organizations:daily-summary", organization):
+            ctx = OrganizationReportContext(timestamp, duration, organization, daily=True)
+            with sentry_sdk.start_span(op="daily_summary.user_project_ownership"):
+                user_project_ownership(ctx)
+            # TODO: convert timezones to UTC offsets and group
+            users_by_tz = defaultdict(list)
+            users_with_tz = user_option_service.get_many(
+                filter=dict(user_ids=list(ctx.project_ownership.keys()), key="timezone")
+            )
+            # if a user has not set a timezone, default to UTC
+            users_without_tz = set(ctx.project_ownership.keys()) - {
+                uo.user_id for uo in users_with_tz
+            }
+            if users_with_tz:
+                users_by_tz["UTC"] = users_without_tz
+            for uo in users_with_tz:
+                users_by_tz[uo.value].append(uo.user_id)
+            for tz in users_by_tz.keys():
+                # Create a celery task per timezone
+                prepare_summary_data.delay(ctx=ctx)
+
+
+@instrumented_task(
+    name="sentry.tasks.daily_summary.prepare_summary_data",
+    queue="reports.prepare",
+    max_retries=5,
+    acks_late=True,
+    silo_mode=SiloMode.REGION,
+)
+@retry
+def prepare_summary_data(
+    ctx: OrganizationReportContext,
+):
+    # build 'Today's Event Count vs. 14 day average'. we need 15 days of data for this
+    fifteen_days = ONE_DAY * 15
+    start = to_datetime(to_timestamp(ctx.end) - fifteen_days)
+    with sentry_sdk.start_span(op="daily_summary.project_event_counts_for_organization"):
+        event_counts = project_event_counts_for_organization(
+            start=start, end=ctx.end, ctx=ctx, referrer=Referrer.DAILY_SUMMARY_OUTCOMES.value
+        )
+        for data in event_counts:
+            project_id = data["project_id"]
+            # project no longer in organization, but events still exist
+            if project_id not in ctx.projects_context_map:
+                continue
+
+            project_ctx = ctx.projects_context_map[project_id]
+            total = data["total"]
+            if data["category"] == DataCategory.ERROR:
+                if data["outcome"] == Outcome.ACCEPTED:
+                    time = datetime.fromisoformat(data["time"])
+                    if time.date() == ctx.end.date():
+                        project_ctx.total_today = total
+                    else:
+                        project_ctx.fourteen_day_total += total
+
+    with sentry_sdk.start_span(op="daily_summary.project_passes"):
+        for project in ctx.organization.project_set.all():
+            project_id = project.id
+            project_ctx = ctx.projects_context_map[project_id]
+            project_ctx.fourteen_day_avg = math.ceil(project_ctx.fourteen_day_total / 14)
```

Comment:
it'd be nice to either make the 14 a constant or perhaps we could make this a variable?